plone/app/layout/globals/context.py

```diff
@@ -163,7 +169,14 @@ def folder(self):
 
     @memoize
     def is_folderish(self):
-        return bool(getattr(aq_base(aq_inner(self.context)), 'isPrincipiaFolderish', False))
+        cond1 = bool(
+            getattr(aq_base(aq_inner(self.context)),
+                    'isPrincipiaFolderish', False))
+        cond2 = False
+        if HAS_PAC:
+            if ICollection.providedBy(self.context):
+                cond2 = True
+        return cond1 or cond2
```

Comment:
To try to reexplain:

With the current plone.app.contenttypes, you can't go on the "contents" tab of a collection.

We were used for a lot of people that the "contents" tab of a collections just list the items that the collection's query bring out.

Thus, you can easily do some managment views, using portal_actions to add a button to push those contents via a post on those. This is easy, and fast, and this is there for years.

The idea is just to have the "contents" tab available, which behaves like today, even if we can't physically add content to a collection, the collection is an hybrid container in the sense that its content is not physically "inside", but it has definitivly a folderish behavior in the sense that we should be able to do actions on the content that it has selected via its criterions.

Problem now, is that on p.a.contenttypes collection, that's not the case without my patches or something similar.